kclvm/runner/src/custom_manifests_data/dict_ignore_none.stdout.golden

```diff
@@ -0,0 +1,4 @@
+age: 10
```

Comment:
After YAML/JSON conversion, the results of the runtime plan lose the original storage order of KCL values, and the code of the runner needs to be reconstructed.

Reply:
> After YAML/JSON conversion, the results of the runtime plan lose the original storage order of KCL values, and the code of the runner needs to be reconstructed.

fixed in #311